celestia-application.md

```diff
@@ -52,6 +52,7 @@ cd $HOME
 rm -rf celestia-app
 git clone https://github.com/celestiaorg/celestia-app.git
 cd celestia-app/
+git checkout 2015a0cac19c841d5971c1155c53fa2b9f736d6b
```

Comment:
@evan-forbes can you confirm this?

Reply:
PS: this commit also contains malleatedTx indexing. Maybe that can be tagged as `0.1.0`.